doc/source/releases/24.2_announce_user.rst

```diff
@@ -10,16 +10,33 @@ Please see the full :doc:`24.2 release notes <24.2_announce>` for more details.
 Highlights
 ===========================================================
 
-Add wizard-like export view for workflow invocations
+A Wizard-like Export View for Workflow Invocations
```

Comment:
I didn't realize my commit was lost (probably on a rebase) https://github.com/galaxyproject/galaxy/commit/82e230bf7c2f40f5cc5b03cf551146c8c1ecb615

Thanks @ahmedhamidawan for writing that up though! And sorry for not noticing the lost commit.

Reply:
Oh no, feel free to replace mine with yours! or we can leave it as is? or .....

Reply:
@davelopez - it's my fault! I rebased 3 days ago and somehow dropped your commit! Sorry about that!

Reply:
No problem! As I said, Ahmed's description is better :smile:
I was just confused for a moment and thought I forgot to write it up but I kind of recalled doing it :sweat_smile: 

Thanks!